subsys/mgmt/hawkbit/hawkbit.c

```diff
@@ -621,7 +661,7 @@ int hawkbit_init(void)
 	LOG_DBG("Action id: current %d", action_id);
 
 	image_ok = boot_is_img_confirmed();
-	LOG_INF("Image is %s confirmed OK", image_ok ? "" : " not");
+	LOG_INF("Image is %s confirmed OK", image_ok ? "" : "not");
```

Comment:
`LOG_INF("Image is %s confirmed OK", image_ok ? "" : "not");`
In case of success, message will contain 2 spaces as in: `Image is**confirmed OK`  
Solution would be:
``LOG_INF("Image is%s confirmed OK", image_ok ? "" : " not");``
Make sense?

Reply:
@sylvioalves You are right, thanks for pointing that out